### ISSUE TITLE
Add Ability to Upload Matches Without TBA

### ISSUE THREAD

**Issue by mazac44** [gafirst/match-uploader#64]
Add the ability to upload match videos for events that are not on FRC Events and TBA.  For example FTC State.  There is an FTC Events API, but this may not always contain the match data.  Being able to upload the match videos without pulling information from an API would be beneficial for this use case.

**Comment by evan10s** [comment]
In this situation, how much match data do you want in the description? Or do you just want to pick "Qualification Match 1," find the relevant files, and upload with a generic description?

**Comment by mazac44** [comment]
Yes it would be a generic description. The score and teams in match part
just wouldn't exist in this case.
